### ISSUE TITLE
Close button in search panel text box is confusing

### ISSUE THREAD

**Issue by gashcrumb** [atlasmap/atlasmap-ui#29]
Just doing some clicking around on staging and noticed the little 'x' doesn't close the search panel after you've clicked the magnifying glass.  Clicking the magnifying glass a second time does close it though.

**Comment by jbakermd** [comment]
Yeah this one is kind of confusing. The little "x" in the text box clears out the search terms, it's not for closing the search. The search icon at the top is what toggles the search box open/closed. 

I think we should have the UI team make a recommendation on this, as we constantly make this mistake in trying to close the box with the little x inside the text box and we've been working with this UI for months.

**Comment by jbakermd** [comment]
Renaming this one for clarification that it's working as designed, but functionality could be designed better.

**Comment by jbakermd** [comment]
@dongniwang fyi.

**Comment by jbakermd** [comment]
One possible solution here is to make the "x" in the text box not only clear the search text, but also close the box / exit search.

**Comment by jbakermd** [comment]
For reference, here's the "x" button we're talking about:

![screenshot 2017-07-19 17 52 31](https://user-images.githubusercontent.com/26657480/28393230-4a3f2000-6cab-11e7-8615-328df33a2978.png)

**Comment by dongniwang** [comment]
@jbakermd @gashcrumb Thanks for bringing this to my attention! 

I was not able to get on to the staging site and see it myself for some reason...

But I did take a look at a PatternFly [live example](https://rawgit.com/patternfly/patternfly/master-dist/dist/tests/table-view.html). See screenshot below as well. I think we're using the PatternFly **[Find](http://www.patternfly.org/pattern-library/forms-and-controls/find/)** pattern. 
![screen shot 2017-07-20 at 10 16 25 am](https://user-images.githubusercontent.com/24943812/28421890-82fe7646-6d34-11e7-82c6-690825fdbb76.png)

Basically, the PatternFly behavior is clicking on both the search icon and the "x" would close the search input box. So I think we can jus follow what PF suggests here. And actually there's no way to clear the text...

cc: @sjcox-rh

**Comment by jbakermd** [comment]
@dongniwang perfect. thanks for the clarification. I'll change the behavior to close the box instead of just clear the text.
